### PR TITLE
feat: schedule

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |y|	
|BC breaks?     |y|	
|Deprecations?  |n|	
|Fixed tickets  |n|	

Job scheduler (CRUD + Job)

## Breaking change
The job emsco:release:publish is not automatically started every seconds

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -56,8 +57,7 @@
 		"symfony/validator": "^4.4",
 		"symfony/web-link": "^4.4",
 		"symfony/webpack-encore-bundle": "^1.0",
-		"symfony/yaml": "^4.4",
-		"ramsey/uuid-doctrine": "^1.6"
```
Comment (Davidmattei) [inline composer.json:60]: why is ramsey/uuid removed?

File: src/Form/Form/ScheduleType.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<?php
+
```
Comment (Davidmattei) [inline src/Form/Form/ScheduleType.php:2]:
```suggestion
declare(strict_types=1);
```

**Review by Davidmattei — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (theus77) [reply composer.json:60]: It was there twice

File: src/Form/Form/ScheduleType.php (hunk 2 above)
Comment (theus77) [reply src/Form/Form/ScheduleType.php:2]: Done: 558ad6dae7580f4fd384745420ee96e9581aaf55

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**